<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feicim File History dialog window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/images/back.png</normaloff>:/icons/images/back.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="nextButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/images/next.png</normaloff>:/icons/images/next.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/images/close.png</normaloff>:/icons/images/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QTableView" name="jobTableView"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableView" name="filesTableView"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
